cypress/integration/manual-aggregated-sync.spec.js

```diff
@@ -1,27 +1,137 @@
-import { dataTest } from "../support/utils";
+import ManualAggregateSyncPageObject from "../pageobjects/ManualAggregateSyncPageObject";
 
+/**
+ * Database: d2-docker-eyeseetea-2-30-datasync-sender
+ */
 context("Manual aggregated sync", function() {
+    const page = new ManualAggregateSyncPageObject(cy);
+
+    const anyOrgUnit = "Ghana";
+    const anyInstance = "Y5QsHDoD4I0";
+    const anyDataset = "Malaria annual data";
+
```

Comment:
Minor: why are those called any? Maybe group in an object.

Reply:
There is no special reason, in Android I usually use constants and call them anyX because the value to understand the test does not matter, ok, I will to remove any keyword and create an object to group variables.

Reply:
Done!